### DISCUSSION TITLE
Click to sign up for the github student pack

### DISCUSSION THREAD

**Discussion by xiaohao8** [community/community#56497] [GitHub Education]
### Hi everyone,

When I signed up for the github student pack, I turned on the GPS and allowed the browser to get location information, but I clicked
continueAt the time.Web page is not responding
video:[https://cloud.06dn.com/s/XX5YF9](url)

**Comment by samus-aran** [comment]
Hi @xiaohao8 👋 

Are you still experiencing this issue?

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍
